core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -610,18 +610,19 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
                   withClonedAuthDomain(sourceAuthDomains.map(n => ManagedGroupRef(RawlsGroupName(n))).toSet, destWorkspaceRequest.authorizationDomain.getOrElse(Set.empty)) { newAuthDomain =>
                     // add to or replace current attributes, on an individual basis
                     val newAttrs = sourceWorkspaceContext.attributes ++ destWorkspaceRequest.attributes
-
-                    withNewWorkspaceContext(destWorkspaceRequest.copy(authorizationDomain = Option(newAuthDomain), attributes = newAttrs, bucketLocation = bucketLocationOption), dataAccess) { destWorkspaceContext =>
-                      dataAccess.entityQuery.copyAllEntities(sourceWorkspaceContext, destWorkspaceContext) andThen
-                        dataAccess.methodConfigurationQuery.listActive(sourceWorkspaceContext).flatMap { methodConfigShorts =>
-                          val inserts = methodConfigShorts.map { methodConfigShort =>
-                            dataAccess.methodConfigurationQuery.get(sourceWorkspaceContext, methodConfigShort.namespace, methodConfigShort.name).flatMap { methodConfig =>
-                              dataAccess.methodConfigurationQuery.create(destWorkspaceContext, methodConfig.get)
+                    traceDBIOWithParent("withNewWorkspaceContext (cloneWorkspace)", parentSpan) { s1 =>
+                      withNewWorkspaceContext(destWorkspaceRequest.copy(authorizationDomain = Option(newAuthDomain), attributes = newAttrs, bucketLocation = bucketLocationOption), dataAccess, s1) { destWorkspaceContext =>
+                        dataAccess.entityQuery.copyAllEntities(sourceWorkspaceContext, destWorkspaceContext) andThen
+                          dataAccess.methodConfigurationQuery.listActive(sourceWorkspaceContext).flatMap { methodConfigShorts =>
+                            val inserts = methodConfigShorts.map { methodConfigShort =>
+                              dataAccess.methodConfigurationQuery.get(sourceWorkspaceContext, methodConfigShort.namespace, methodConfigShort.name).flatMap { methodConfig =>
+                                dataAccess.methodConfigurationQuery.create(destWorkspaceContext, methodConfig.get)
+                              }
                             }
-                          }
-                          DBIO.seq(inserts: _*)
-                        } andThen {
-                        DBIO.successful((sourceWorkspaceContext, destWorkspaceContext))
+                            DBIO.seq(inserts: _*)
+                          } andThen {
+                          DBIO.successful((sourceWorkspaceContext, destWorkspaceContext))
+                        }
```

Comment:
github's diff is really strange. the change here is basically a one liner
![Screen Shot 2021-03-03 at 6 52 30 PM](https://user-images.githubusercontent.com/5375000/109888723-f9cba480-7c51-11eb-8dd2-f699df70477f.png)